### PR TITLE
Updating to lexibank 2.1.

### PR DESCRIPTION
**Pull request by SimonGreenhill**
@LinguList -- not sure why the changes to forms.csv, but everything else looks ok?

### CODE REVIEW

**Review by LinguList — APPROVED** [review]
I think it's just an ordering problem, although it is not clear to me, why this happens now. Is it the Python version?

**Comment by LinguList** [comment]
BTW, I noticed, in your reviews, @SimonGreenhill, that you 

1. imported the Concept as BaseConcept, and
2. replaced the `langauge|concept_lookup` by `language` and `concept`, respectively.

I am fine with this, and we do not, maybe, need to discuss this for too long, but what about saying that we really use these when coding? My name-choice was so far: CustomConcept and the lookup-constructs. But I could switch then. And I think everybody should then also do so.

I assume, the more this is similar, the easier it is to debug later, right?

**Comment by LinguList** [comment]
This involves also @tresoldi and @chrzyki

**Comment by chrzyki** [comment]
I have no strong preferences here but have tried to stick to `CustomConcept`, `CustomLanguage` etc. I have not been perfectly adamant with the `languages`, `language_lookup` etc. to be honest but I agree that uniformity here is good.

**Comment by SimonGreenhill** [comment]
Yes, we should be consistent. I was using BaseX overridden as X, but I'm happy to change to CustomX. 

Re: `x_lookup`, I much prefer x as it's less to type, and cleaner.

File: cldf/cldf-metadata.json [hunk 1]
```diff
@@ -18,19 +18,19 @@
...
+            "dc:created": "4bc6519"
         },
         {
             "rdf:type": "prov:Entity",
-            "dc:title": "Concepticon",
-            "rdf:about": "https://github.com/concepticon/concepticon-data/",
-            "dc:created": "v2.1.0-109-g408e9f8"
+            "dc:title": "Glottolog",
+            "rdf:about": "https://github.com/glottolog/glottolog",
+            "dc:created": "v4.0-56-g185d304cba"
```
Comment (chrzyki) [inline cldf/cldf-metadata.json]: Please specify the version explicitly with --glottolog-version.

File: cldf/cldf-metadata.json [hunk 2]
```diff
@@ -18,19 +18,19 @@
...
         },
         {
             "rdf:type": "prov:Entity",
-            "dc:title": "Glottolog",
-            "rdf:about": "https://github.com/lingulist/glottolog-data",
-            "dc:created": "v4.0"
+            "dc:title": "Concepticon",
+            "rdf:about": "https://github.com/concepticon/concepticon-data",
+            "dc:created": "v2.2.0-4-gaadba44"
```
Comment (chrzyki) [inline cldf/cldf-metadata.json]: Likewise with --concepticon-version

Comment (SimonGreenhill) [reply cldf/cldf-metadata.json]: fixed

File: cldf/cldf-metadata.json (hunk 1 above)
Comment (SimonGreenhill) [reply cldf/cldf-metadata.json]: fixed

**Comment by LinguList** [comment]
Okay, so concepts, languages, are now our lookups. If I port my data further, I'll adjust this. Can we add this to our style guide? I mean: we say, we usually need a concept lookup, which we call "concepts", and a language lookup, which we call "languages"?

**Pull request merged**